starter-code/src/app/components/log-form/log-form.component.ts

```diff
@@ -0,0 +1,35 @@
+import { Component, OnInit } from '@angular/core';
+import { AccessControlLogService } from '../../services/access-control-log.service';
+
+@Component({
+  selector: 'app-log-form',
+  templateUrl: './log-form.component.html',
+  styleUrls: ['./log-form.component.css']
+})
+export class LogFormComponent implements OnInit {
+  feedbackEnabled = false;
+  processing = false; 
+  formSendCorrectly = false;
+
+  person : string = '';
+  message : string = '';
+
+  constructor(private accessControlLogService: AccessControlLogService) { }
+
+  ngOnInit() {
+  }
+
+  handleClick(form) {
+    if (form.valid) {
+      this.person = form.value.person;
+      this.message = form.value.message;
```

Comment:
You are two-way binding name and message in your form, therefore you don't need lines 24 and 25.